### DISCUSSION TITLE
Need help getting Overlays in SVG to display

### DISCUSSION THREAD

**Discussion by vlaryn** [ExperienceLovelace/ha-floorplan#168] [Q&A] [answered]
I am trying to get HA-Floorplan setup using a tutorial by [Bitbon](https://www.youtube.com/watch?v=MCNxgb0mrSA&t=5476s) on Youtube. I am following along and all works up to the point where you are able to click on an area in your SVG file and it change the Style in the SVG file to blend in another image. This is the effect of a light being turned on in the room. I get no errors in Chrome inspector until I add the section 
`              state_action:
                action: 'call-service'
                service: floorplan.style_set
                service_data:
                  element: "${entity.entity_id.replace('light.', 'light_overlay.')}"`
in my yaml file. Please see my code in links below. 

Thanks for any help

floorplan_optimized.svg - https://paste-bin.xyz/16128
floorplan2.css - https://paste-bin.xyz/16126
floorplan2.yaml - https://paste-bin.xyz/16124

**Comment by OzGav** [comment]
OK firstly that guys floorplan is EPIC!!!!!!!!!!!!!!!!!!!!!!!!!

Now on to you. You will need to provide some more info about the error you are seeing. Having said that what that video is showing you is really advanced. You might be better served starting with something more simple. Do you have a background image for your house or an example? What are you trying to do?

I opened your SVG in Inkscape but because it has links to other files it wasn't working properly.

**Reply by vlaryn** [reply]
Well I found my issues. I was missing a "  `  " at the end of my js code that was being using in the element section. I might have had issues in the SVG file as well. I am not sure as I totally redid the file during my troubleshooting.

**Comment by vlaryn** [comment, marked as answer]
Well I found my issues. I was missing a " ` " at the end of my js code that was being using in the element section. I might have had issues in the SVG file as well. I am not sure as I totally redid the file during my troubleshooting.
